`has-bigints` is not included as a dependency

`has-bigints` was added as a dependency in https://github.com/inspect-js/is-bigint/commit/7bb9d7ab42214c12ce25e9f0cfe2af769388c3bb; however, the dependency was not added to `package.json`.

In a script like this:
```js
require('is-bigint')
```

I get the following error:
```
node index.js
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'has-bigints'
Require stack:
- /Users/marc/projects/zzz/is-bigint/node_modules/is-bigint/index.js
```

--- a/package.json
+++ b/package.json
@@ -33,12 +33,14 @@
 		"url": "https://github.com/inspect-js/is-bigint/issues"
 	},
 	"homepage": "https://github.com/inspect-js/is-bigint#readme",
+	"dependencies": {
+		"has-bigints": "^1.0.1"
+	},
 	"devDependencies": {
 		"@ljharb/eslint-config": "^17.6.0",
 		"aud": "^1.1.5",
 		"auto-changelog": "^2.3.0",
 		"eslint": "^7.32.0",
-		"has-bigints": "^1.0.1",
 		"has-symbols": "^1.0.2",
 		"nyc": "^10.3.2",
 		"object-inspect": "^1.11.0",

